# Header
std_msgs/Header header

#Data
uint8 num_people
person[] people
sensor_msgs/Image color
sensor_msgs/Image depth